flink-tests/src/test/java/org/apache/flink/test/checkpointing/AbstractEventTimeWindowCheckpointingITCase.java

```diff
@@ -115,6 +117,14 @@ public static void stopTestCluster() {
 	@Before
 	public void initStateBackend() throws IOException {
 		switch (stateBackendEnum) {
+			case MEM_ASYNC:
+				this.stateBackend = new AsyncMemoryStateBackend(MAX_MEM_STATE_SIZE);
+				break;
+			case FILE_ASYNC: {
+				String backups = tempFolder.newFolder().getAbsolutePath();
+				this.stateBackend = new AsyncFsStateBackend("file://" + backups);
```

Comment:
To make this work cross platform, always use `file.toUri()` or `new Path(file.toUri())`.